integration/oltp_e2e/config/config.yaml

```diff
@@ -0,0 +1,19 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+
+processors:
+  gnmipath:
+
+exporters:
+  # NOTE: Prior to v0.86.0 use `logging` instead of `debug`.
+  gnmi:
+
+service:
+  pipelines:
+    metrics:
+      receivers: [otlp]
```

Comment:
do we need the docker receiver here?

Reply:
I don't think so, the otlp_e2e test is the one that was previously in the root directory. It's "only" checking whether otel metrics can be passed through/transformed via clio successfully. The oltp_e2e and docker_stats_e2e intrgration tests are complementary rather than inclusive.